src/Symfony/Component/Console/Application.php

```diff
@@ -156,6 +140,39 @@ public function run(InputInterface $input = null, OutputInterface $output = null
     }
 
     /**
+     * Handler command run exception and provides with the exit code in return
+     *
+     * @param \Exception|\Throwable $e
+     * @param OutputInterface $output
+     * @return int|mixed
+     * @throws \Throwable
+     */
+    private function handleCommandRunException($e, OutputInterface $output)
+    {
+        if (!$this->catchExceptions) {
+            throw $e;
+        }
+
+        if ($output instanceof ConsoleOutputInterface) {
+            $this->renderException($e, $output->getErrorOutput());
+        } else {
+            $this->renderException($e, $output);
```

Comment:
The phpdoc of `renderException` with `@param \Exception      $e` is not correct anymore and needs to be updated too.